lib/exercises.rb

```diff
@@ -2,19 +2,46 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n)
+# Space Complexity: O(n)
 
 def grouped_anagrams(strings)
```

Comment:
👍 This works and assuming the words are small you are right on the time/space complexity.